.pa11yci

```diff
@@ -7,11 +7,15 @@
     "http://localhost:8000/audit"
   ],
   "defaults": {
+    "timeout": 60000,
     "standard": "WCAG2AA",
     "runners": [
       "htmlcs",
       "axe"
     ],
-    "hideElements": ".pa11y-ignore, .usa-select, .usa-step-indicator__segment-label, .is-hidden, #upload-worksheet, .usa-file-input__instructions, #my-submissions"
+    "hideElements": ".pa11y-ignore, .usa-select, .usa-step-indicator__segment-label, .is-hidden, #upload-worksheet, .usa-file-input__instructions, #my-submissions",
+    "chromeLaunchConfig": {
+      "executablePath": "/usr/bin/google-chrome"
+    }
```

Comment:
After a whole bunch of tests, it turned out pa11y was silently failing to launch headless chrome, then timing out. I couldn't pinpoint the exact source of this change in the dependency tree, but it turns out we just needed to point it to chrome's path.

Ref: https://github.com/pa11y/pa11y-ci/issues/198